build.sh

```diff
@@ -1,89 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I'm not sure if removing the Makefile and build.sh is a good idea. It's the only way to run the Docker build locally. :thinking:

Reply:
I put them back.

Reply:
Thanks